invenio_oaiserver/query.py

```diff
@@ -45,6 +46,23 @@ def match(self, record):
         return self.query.accept(MatchUnit(record))
 
 
+def get_affected_records(spec=None, search_pattern=None):
+    """Get list of affected records."""
+    sets = '_oai.sets:{0}'.format(spec) if spec else None
+    search_pattern = search_pattern if search_pattern else None
+
+    query = " OR ".join(filter(None, [sets, search_pattern]))
+    body = {'query': Query(query).query.accept(ElasticSearchDSL())}
+
+    response = current_search_client.search(
```

Comment:
It comes from `elasticsearch.helpers`, are you sure that insert a direct dependency of `elasticsearch-py` here is a good idea? Can I access from `invenio-search`?